<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserManagementDialog</class>
 <widget class="QDialog" name="UserManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управление персоналом</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidgetUsersPermissions">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="Users">
      <attribute name="title">
       <string>Персонал</string>
      </attribute>
      <widget class="QGroupBox" name="groupBoxUserInfo">
       <property name="geometry">
        <rect>
         <x>402</x>
         <y>9</y>
         <width>361</width>
         <height>521</height>
        </rect>
       </property>
       <property name="title">
        <string>Информация о сотруднике:</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QGroupBox" name="groupBoxUserCredentials">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>171</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string>Данные для входа в систему:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>54</y>
           <width>135</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="labelPassword">
            <property name="toolTip">
             <string>Имя пользователя для входа в систему</string>
            </property>
            <property name="text">
             <string>Пароль:</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>13</y>
           <width>135</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelLogin">
            <property name="toolTip">
             <string>Имя пользователя для входа в систему</string>
            </property>
            <property name="text">
             <string>Имя пользователя:</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>100</y>
           <width>135</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="labelPassword_2">
            <property name="toolTip">
             <string>Имя пользователя для входа в систему</string>
            </property>
            <property name="text">
             <string>Подтверждение пароля:</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_3</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBoxUniqData">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>135</width>
          <height>95</height>
         </rect>
        </property>
        <property name="title">
         <string>Персональные данные:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="labelName">
             <property name="text">
              <string>ФИО:</string>
             </property>
             <property name="buddy">
              <cstring>lineEdit</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="labelPhone">
             <property name="text">
              <string>Телефон:</string>
             </property>
             <property name="buddy">
              <cstring>lineEdit_2</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBoxSalary">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>347</y>
          <width>189</width>
          <height>97</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>:)</string>
        </property>
        <property name="title">
         <string>Настройки зарплаты:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="labelRate">
             <property name="text">
              <string>Ставка с работы</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxRate</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxRate"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="labelPercent">
             <property name="text">
              <string>% с работы:</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxPermissions</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxPermissions"/>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="labelSPday">
             <property name="text">
              <string>Ставка в день</string>
             </property>
             <property name="buddy">
              <cstring>spinBoxSPday</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxSPday"/>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="labelSalePercent">
             <property name="text">
              <string>Процент с продаж:</string>
             </property>
             <property name="buddy">
              <cstring>spinBox_3</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox_3"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBoxUserList">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>276</width>
         <height>278</height>
        </rect>
       </property>
       <property name="title">
        <string>Список сотрудников:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonWorking">
            <property name="text">
             <string>Работающие</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonFired">
            <property name="text">
             <string>Уволенные</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelFilter">
            <property name="text">
             <string>Фильтр:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="FilterLineEdit" name="lineEditFilter"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QTreeView" name="treeViewUsers">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="indentation">
           <number>-1</number>
          </property>
          <attribute name="headerCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Permissions">
      <attribute name="title">
       <string>Права доступа</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGroups">
         <property name="minimumSize">
          <size>
           <width>191</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Группы:</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonAddGroup">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QListView" name="listViewGroups">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonDeleteGroup">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxPermissions">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Права:</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="PermissionsWidget" name="widgetPermissions" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>297</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PermissionsWidget</class>
   <extends>QWidget</extends>
   <header>permissionswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>filterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
